extensions/sql-database-projects/src/test/baselines/openNewStyleSqlProjectWithFilesSpecifiedBaseline.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build">
+  <Sdk Name="Microsoft.Build.Sql" Version="1.0.0" />
+  <PropertyGroup>
+    <Name>TestProjectName</Name>
+    <ProjectGuid>{2C283C5D-9E4A-4313-8FF9-4E0CEE20B063}</ProjectGuid>
+    <DSP>Microsoft.Data.Tools.Schema.Sql.Sql150DatabaseSchemaProvider</DSP>
+    <ModelCollation>1033, CI</ModelCollation>
+  </PropertyGroup>
+  <Target Name="BeforeBuild">
+    <Delete Files="$(BaseIntermediateOutputPath)\project.assets.json" />
+  </Target>
+  <ItemGroup>
+    <Folder Include="Properties" />
+    <Folder Include="folder1" />
+  </ItemGroup>
+  <ItemGroup>
+    <Build Include="folder1\file2.sql" />
+    <Build Include="file1.sql" />
+    <Build Include="..\Test\Test.sql" />
```

Comment:
Is there also a test scenario for `<Build Remove="sql" />

Reply:
I'm planning on adding support for Build Remove excluded items in a separate PR to keep these PRs smaller (mentioned it in the description)